opensrp-reveal/src/main/assets/json.form/bednet_distribution.json

```diff
@@ -353,7 +353,7 @@
         "openmrs_entity": "",
         "openmrs_entity_id": "",
         "type": "edit_text",
-        "hint": "How many of these nets are normal nets and have been dipped in the last 12 months?",
+        "hint": "How many are normal nets that were dipped in the last 6 months?",
```

Comment:
@githengi  this field has a default value of 0 which makes the label shift up. When the label shifts up above the value, we loose the last half of the text. Currently it only displays "How many are normal nets that were d" How can I change the size of this text? Should I use a label?

Reply:
This was resolved by using a shortened hint once a value is provided